src/node/core.go

```diff
@@ -67,12 +69,21 @@ func NewCore(id uint64, key *ecdsa.PrivateKey, participants *peers.Peers,
 		logEntry = logger.WithField("id", id)
 	}
 
+	// add some creation rates for node simulation
+	evCreationRate := 1.0
+
+	if id % 3 == 0 {
+		//evCreationRate = 0.05
+	}
+	logEntry.Debug("ZZZZ core id=", id, "has creation ratio", evCreationRate, "\n")
```

Comment:
logEntry already has Field("id", id).
"ZZZZ" is nice for quick debug, not for master.